src/lib/Scenes/Show/Screens/Detail.tsx

```diff
@@ -69,11 +69,11 @@ export class Detail extends React.Component<Props, State> {
       },
     })
 
-    if (show.location && !isEmpty(show.location.displayDaySchedules)) {
+    if (show.location && !isEmpty(show.location)) {
```

Comment:
Why isn't the truthy check for show.location sufficient here/do we need the isEmpty() call? Or does the LocationMap_location sometimes return an empty object?

Reply:
Good question!

Reply:
good point, i removed the `displayDaySchedule` part and i think this doesn't make much sense anymore, i'll do a small follow up PR and merge on green it. 

Thanks both for looking